newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackPurchaseDialog.js

```diff
@@ -0,0 +1,291 @@
+// @flow
+import * as React from 'react';
+import { Trans } from '@lingui/macro';
+import {
+  listUserPurchases,
+  type PrivateAssetPackListingData,
+} from '../../Utils/GDevelopServices/Shop';
+import Dialog, { DialogPrimaryButton } from '../../UI/Dialog';
+import AuthenticatedUserContext from '../../Profile/AuthenticatedUserContext';
+import CreateProfile from '../../Profile/CreateProfile';
+import Text from '../../UI/Text';
+import { useInterval } from '../../Utils/UseInterval';
+import { getStripeCheckoutUrl } from '../../Utils/GDevelopServices/Shop';
+import Window from '../../Utils/Window';
+import { Line, Spacer } from '../../UI/Grid';
+import CircularProgress from '../../UI/CircularProgress';
+import BackgroundText from '../../UI/BackgroundText';
+import { showErrorBox } from '../../UI/Messages/MessageBox';
+import VerifiedUser from '@material-ui/icons/VerifiedUser';
```

Comment:
Should we use one of our icons?

<img width="61" alt="image" src="https://user-images.githubusercontent.com/32449369/197165106-252c1e80-bf22-4ed5-80ce-0d861804ebad.png">

Reply:
Not sure it's the same intent though

Reply:
gonna use a simple "Mark"